scripts/hid_configurator/configurator.py

```diff
@@ -1,151 +1,231 @@
+import os
 import hid
 import struct
 import time
-from enum import Enum
 
 import argparse
 import logging
```

Comment:
Maybe we could decide to use logging or print, instead of mixing both?

Reply:
logging is for debug purposes, print is for interaction with user; I don't see the problem with using both

Reply:
ok